### ISSUE TITLE
How to connect to localhost confluence instance? http://localhost:1990/confluence

### ISSUE THREAD

**Issue by pitikdmitry** [leanconvert/create-react-app-atlassian-connect-addon#3]
I set "atlassian-connect-dev-instance-host": "http://localhost:1990/confluence" in package.json, but when i load add_on url in confluence it shows me error:
Could not install the file https://005966d2.ngrok.io/atlassian-connect.json. Check that the file is valid. 
![image](https://user-images.githubusercontent.com/22732026/56023930-a4addf00-5d17-11e9-8ea6-74ea92b88229.png)

**Comment by vitaliysobur** [comment]
Hi @pitikdmitry, can you please confirm you use the **cloud** version of Confluence?

**Comment by pitikdmitry** [comment]
Hi. If I use cloud version all is OK. I started local instance of confluence with atlas-run and I can’t connect react add on to it.

**Comment by vitaliysobur** [comment]
As far as I know atlas-run spins up a server instance on your local machine. So it is different from the cloud one. That is why the add-on fails to install

**Comment by pitikdmitry** [comment]
Yes you are right, but why i cant connect to local instance? What is the difference between connecting to the local instance and the cloud instance?

**Comment by vitaliysobur** [comment]
This addon uses atlassian connect framework that is supported by the cloud instances only.  Server instances use different approach / framework for building Addons and usually built in Java

**Comment by pitikdmitry** [comment]
thanks for the reply, do you know how to connect react Addon to the server instance?
